File: id/src/main/java/com/simprints/id/tools/Log.java
package com.simprints.id.tools;

import java.util.Locale;

@SuppressWarnings("unused")
public class Log {

    public static void d(Object o, String s) {
        android.util.Log.d(o.getClass().getSimpleName(),
                String.format(Locale.UK, "%s: %s", o.getClass().getSimpleName(), s));
    }
}

File: id/src/main/java/com/simprints/id/tools/Log.kt
package com.simprints.id.tools

import timber.log.Timber
import java.util.*

object Log {

    fun d(o: Any, s: String) {
        Timber.d(String.format(Locale.UK, "%s: %s", o.javaClass.simpleName, s))
    }
}
